lib/elixir_google_scraper_web/controllers/keyword_controller.ex

```diff
@@ -26,4 +28,16 @@ defmodule ElixirGoogleScraperWeb.KeywordController do
     |> put_flash(flash_type, flash_message)
     |> redirect(to: Routes.keyword_path(conn, :index))
   end
+
+  def show(conn, %{"id" => id}) do
+    keyword = Repo.get_by(Keyword, %{id: id})
```

Comment:
How about a function in `Keywords` context?

Reply:
If we move this function to context, we can move preload function below to the same function. Maybe this work
```elixir
Keyword
|> preload(:search_result)
|> get(id)
```